include/fw_info.rst

```diff
@@ -29,6 +29,9 @@ It includes the following information:
 * The single, monotonically increasing version number of the image.
 * The address through which to boot into the firmware (the vector table address).
   This address is not necessarily the start of the image.
+* A value that can be modified in place to invalidate the firmware. See :option:`CONFIG_FW_INFO_VALID_VAL`.
+  If this is set to any other value than :option:`CONFIG_FW_INFO_VALID_VAL` (e.g. by the bootloader), the image can quickly be established as invalid.
+  The bootloader will set this to 0 when the image fails validation in some way, so that there is no need to perform a costly validation on every boot.
```

Comment:
Maybe add a point on this value being guarded from denial-of-service invalidation by a third party

Reply:
Added a sentence